tools/llm_review.py

```
@@ -889,19 +962,28 @@ async def tool_function(**kwargs):
             if client is not None:
                 try:
                     await client.disconnect()
-                except Exception:
+                except BaseException:
                     pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)